particles/Renderer.cpp

```diff
@@ -312,12 +337,10 @@ void Renderer::init(score::gfx::RenderList& renderer)
                 QRhiCommandBuffer& cb,
                 score::gfx::Edge& edge)
     {
-        auto& n = static_cast<const Node&>(this->node);
-        //auto mesh = n.mesh;
-        const TexturedMeshForParticles mesh(n.meshName);
-        /*n.mesh.myData = newmesh.myData;
-        n.mesh.mesh = newmesh.mesh;
-        auto& mesh = n.mesh;*/
+        // Don't render when there's no data to avoid crashes
+        if(!m_meshBuffer || m_mesh.vertexCount == 0)
```

Comment:
Don't try to render particles if the mesh hasn't loaded for some reason as this can crash